### PR TITLE
Fix AngularCli integration issues

### PR DESCRIPTION
**Pull request by stevehouel**
Fix integration issues with AngularCli regarding styling, polyfills and content loading.
Also add default proxy configuration when using ng serve instead of yarn start. User can now run `ng serve —proxy-conf proxy.conf.json` to use his backend.

Related to #5558

### CODE REVIEW

File: generators/client/templates/angular/_.angular-cli.json [hunk 1]
```diff
@@ -32,25 +42,36 @@
     }
   },
   <%_ } _%>
+  "lint": [
+    {
+      "project": "../../../tsconfig.json"
+    },
+    {
+      "project": "../../../tsconfig.-aotjson"
```
Comment (hdurix) [inline generators/client/templates/angular/_.angular-cli.json]: Misspelled `tsconfig-aot.json` no?

Comment (stevehouel) [inline generators/client/templates/angular/_.angular-cli.json]: Yes Typo issue :)

Comment (hdurix) [inline generators/client/templates/angular/_.angular-cli.json]: The '.' is missing now ;)

Comment (stevehouel) [inline generators/client/templates/angular/_.angular-cli.json]: You know I'm a bit sick those days lol :)

Comment (hdurix) [inline generators/client/templates/angular/_.angular-cli.json]: Take vitamins for the end of the week, you'll need it!

**Comment by kuncevic** [comment]
does that fully support angular-cli ? Say if I modify custom `scripts` or `assets` section of `.angular-cli.json` will it all work then?

**Pull request merged**